### ISSUE TITLE
Menu configuration

### ISSUE THREAD

**Issue by Anthony-the-second** [thelink2012/modloader#81]
Hello, i've recently installed ModLoader (with cleo 4.3 and silent asi loader). Mod work when i put it in the mod folder but, the "menu configuration" doesn't appear in my GTA SA option 
<a href="http://zupimages.net/viewer.php?id=19/11/24ce.png"><img src="https://zupimages.net/up/19/11/24ce.png" alt="" /></a>  

(I'm new here and i have a bad english, sorry if i do something wrong)

**Comment by ipwwn** [comment]
Having this issue on my laptop installation as well, for some reason its working fine on my desktop PC, I believe it has something to do with modloader not being fully and truly portable, as I've found out it creates several folders on local  AppData and ProgramData , also it does weird stuff to the registry apparently because Uninstalling san andreas with Revo Uninstaller (gets rid of left over files and registry keys) and re installing with a fresh copy of modloader is the only method Ive found that works. Also sometimes the game would "fail" to re create the inis in the modloader folder.

**Comment by PRinceBhai** [comment]
Just delete Modloader's app data in %appdata%.

**Comment by MiranDMC** [comment]
When using **Ultimate ASI Loader** make sure it is named **vorbisFile.dll**. With other names it works and loads ASI files, but ModLoader won't work correctly or at all.

**Comment by Oussama-Laaziz07** [comment]
Yeah i had the same problem with my laptop aswell,when i searched i found out that my laptop did not copy all the files that i downloaded from modloader folder,so just copy the config,modloader ini,and text folder from the modloader u installed to Gta san andreas>modloader>.data 
It took me a while but it worked for me.
